pkgs/development/python-modules/sphinxcontrib-moderncmakedomain/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildPythonPackage,
+  defusedxml,
+  fetchPypi,
+  hatchling,
+  pytestCheckHook,
+  sphinx,
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-moderncmakedomain";
+  version = "3.29.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "sphinxcontrib_moderncmakedomain";
+    hash = "sha256-NYfe8kH/JXfQu+8RgQoILp3sG3ij1LSgZiQLXz3BtbI=";
+  };
+
+  nativeBuildInputs = [
+    hatchling
+    sphinx
+  ];
+
+  nativeCheckInputs = [
+    defusedxml
+    pytestCheckHook
+    sphinx
+  ];
+
+  pythonNamespaces = [ "sphinxcontrib" ];
+
```

Comment:
Could you enable `pythonImportsCheck`?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck-using-pythonimportscheck

Reply:
Sounds good. Thanks for the feedback